app/src/main/java/com/coveros/coverosmobileapp/oauth/example/OAuthLoginActivity.java

```diff
@@ -61,6 +61,7 @@ protected void onCreate(Bundle savedInstanceState) {
      * @param webView    WebView for which the WebViewClient is set
      * @param authCallback    callback to wait for the parsing of the authorization code from the redirect uri
      */
+    @Deprecated
```

Comment:
Remove this and the suppress warning for deprecation and use a non deprecated method